### PR TITLE
Update to Qt5 syntax and usage.

### PR DESCRIPTION
**Pull request by beroset**
Signed-off-by: Ed Beroset <beroset@ieee.org>

### CODE REVIEW

File: services/batteryservice.cpp [hunk 1]
```diff
@@ -43,8 +43,8 @@ quint8 BatteryService::level()
 void BatteryService::onCharacteristicChanged(const QLowEnergyCharacteristic &c, const QByteArray &value)
 {
     if (c.uuid() == QBluetoothUuid(QString(BATTERY_LVL_UUID))) {
-        const quint8 *data = reinterpret_cast<const quint8 *>(value.constData());
-        quint8 val = data[0];
+        //const quint8 *data = reinterpret_cast<const quint8 *>(value.constData());
```
Comment (FlorentRevest) [inline services/batteryservice.cpp]: You can just remove this line

**Review by FlorentRevest — APPROVED** [review]

**Comment by FlorentRevest** [comment]
The title of the PR says Qt5, I guess you meant Qt6 ?

**Comment by FlorentRevest** [comment]
I see you added a new commit to address the PR review. I recommend reading https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d for an introduction about git interactive rebases. This will help you edit commits you already created and keep a clean git history.

In this specific case, it's kinda acceptable to put everything in one commit and the github PR UI has a button that lets me squash all of your commits into one. But in other contributions, it's possible that we'd need to keep multiple commits but in a clean log. :) 

Thank you!

**Comment by beroset** [comment]
Thanks for the hint on interactive rebase.  I know it wasn't quite right but didn't know how to fix it.  Now I do!

**Pull request merged**